packages/paste-core/components/disclosure/src/DisclosureHeading.tsx

```diff
@@ -56,6 +60,7 @@ const StyledDisclosureHeading = React.forwardRef<HTMLDivElement, StyledDisclosur
         >
           <Box
             as="span"
+            element={`${element}_ICON`}
```

Comment:
There's a story for icon (which is weird if you had to add this @TheSisb, was it actually working?) but no test to make sure we don't accidentally remove it.

Can we add a test for DISCLOSURE_HEADING_ICON?

Reply:
I just added the story for icon, there wasn't one before.
I'll add a test.